### DISCUSSION TITLE
markdown标题列表位置调整

### DISCUSSION THREAD

**Discussion by raysun0501** [vuepress-theme-hope/vuepress-theme-hope#3327] [Q&A] [answered]
<img width="1029" alt="image" src="https://github.com/vuepress-theme-hope/vuepress-theme-hope/assets/42628912/deedd65e-75ac-42c2-b4cd-ff75cb9421e4">
有什么方法，可以通过toc配置或者其他方法将markdown的标题列表展示在左侧而非默认的右侧吗？或者说有什么方法可以将目录展示在文章的最上方吗？

**Comment by Mister-Hope** [comment, marked as answer]
自定义组件
